package.json

```
@@ -13,19 +13,20 @@
   },
   "dependencies": {
     "@tanstack/react-query": "~5.53.3",
-    "lucide-react": "~0.417.0",
-    "next": "~14.2.10",
+    "lucide-react": "^0.417.0",
+    "next": "^15.1.4",
```

```suggestion
    "next": "^15.1.4",
```
downgrade back to next": "~14.2.10" next 15 is currently really unstable